src/Logging/DevHome.Logging.csproj

```diff
@@ -5,7 +5,7 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
     <Platforms>x86;x64;arm64</Platforms>
-    <RuntimeIdentifiers>win-x86;win-x64;win-arm64</RuntimeIdentifiers>
+    <RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>

```

Comment:
These need to have the `10` remove, ya?

Reply:
Per https://github.com/microsoft/devhome/pull/2156, the RidGraph workaround would crash the extension, as it meant "use old runtime identifier system instead of new one" instead of "work with packages still using old win10 runtime identifiers. 

Has their been a newer update/interpretation to this workaround?